bucket/amidst.json

```diff
@@ -1,23 +1,19 @@
 {
-    "homepage": "https://github.com/toolbox4minecraft/amidst",
-    "description": "Minecraft interface and data/structure tracking tool",
     "version": "4.6",
+    "description": "Minecraft interface and data/structure tracking tool",
+    "homepage": "https://github.com/toolbox4minecraft/amidst",
     "license": "GPL-3.0-only",
-    "url": "https://github.com/toolbox4minecraft/amidst/releases/download/v4.6/amidst-v4-6.jar#/amidst.jar",
-    "hash": "137f4f718815f04a933325243ddca126c1d8d7720dcf8026926678a8a1ebe65b",
-    "bin": "amidst.bat",
+    "url": "https://github.com/toolbox4minecraft/amidst/releases/download/v4.6/amidst-v4-6.exe#/amidst.exe",
+    "hash": "8b75f93aca5c73074d3f1578d9251a4d05bbafcac3dd0d4e68b2d499eded6305",
+    "bin": "amidst.exe",
     "shortcuts": [
         [
-            "amidst.bat",
+            "amidst.exe",
             "Amidst"
         ]
     ],
-    "pre_install": "Set-Content -Path \"$dir\\amidst.bat\" -Value \"pushd $dir && javaw -jar amidst.jar && popd\"",
-    "suggest": {
-        "Java Runtime Environment": "java/adoptopenjdk-hotspot-jre"
-    },
```

Comment:
This part is still needed since the executable doesn't bundle a JRE. (If it did, it would be 60+ MB.)

Reply:
Good catch, didn't realise that 👍